tests/opwen_email_server/integration/test_celery.py

```diff
@@ -38,3 +40,13 @@ def test_send_message(self):
                 routing_key=self.routing_key,
                 declare=[self.queue],
             )
+
+    def test_celery_task_routes(self):
+        registered_celerey_tasks = [
+            task_name for task_name in current_app.tasks.keys() if not task_name.startswith('celery.')
+        ]
+
+        for task_name in registered_celerey_tasks:
+            self.assertIs(
```

Comment:
You can use [assertIn](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn) to make the resulting error message automatically more meaningful without having to use a custom error message.